### PR TITLE
Update SPOP (with COUNT) tests for Redis 6

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

**Comment by sazzad16** [comment]
@gkorland @marcosnils ping!

File: src/test/java/redis/clients/jedis/tests/commands/SetCommandsTest.java [hunk 1]
```diff
@@ -142,34 +145,46 @@ public void spop() {
...
+    Set<byte[]> bsuperSet = new HashSet<byte[]>();
+    bsuperSet.add(bc);
+    bsuperSet.add(bb);
+    bsuperSet.add(ba);
 
     Set<byte[]> bmembers = jedis.spop(bfoo, 2);
 
     assertEquals(2, bmembers.size());
-    assertByteArraySetEquals(bexpected, bmembers);
+    assertByteArrayCollectionContainsAll(bsuperSet, bmembers);
```
Comment (gkorland) [inline src/test/java/redis/clients/jedis/tests/commands/SetCommandsTest.java]: Why not having a simple comparison and make sure we're getting back the exact two elements we expect?

File: src/test/java/redis/clients/jedis/tests/commands/SetCommandsTest.java [hunk 2]
```diff
@@ -142,34 +145,46 @@ public void spop() {
...
+    Set<String> superSet = new HashSet<String>();
+    superSet.add("c");
+    superSet.add("b");
+    superSet.add("a");
 
     Set<String> members = jedis.spop("foo", 2);
 
     assertEquals(2, members.size());
-    assertEquals(expected, members);
+    assertCollectionContainsAll(superSet, members);
```
Comment (gkorland) [inline src/test/java/redis/clients/jedis/tests/commands/SetCommandsTest.java]: Why not having a simple comparison and make sure we're getting back the exact two elements we expect?

Comment (sazzad16) [reply src/test/java/redis/clients/jedis/tests/commands/SetCommandsTest.java]: From SPOP, we don't know the exact elements we're going to get if fewer elements are requested.

**Pull request merged**